kosmorrolib/model.py

```diff
@@ -239,6 +239,21 @@ def __repr__(self):
             self.details,
         )
 
+    def __hash__(self) -> int:
+        return hash(
+            self.event_type, self.objects, self.start_time, self.end_time, self.details
+        )
```

Comment:
Whoops! I left that in on accident while I was trying to make `found_events` a set instead of a list so duplicate events would be ignored. If I understood correctly, both the `__eq__` and `__hash__` methods are required to check if an object is already in a set. I will remove it since it isn't being used. I'm still very new to Python in general too 😁